firefox-ios/EcosiaTests/Account/Auth/AuthStateActionTests.swift

```diff
@@ -0,0 +1,369 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Common
+@testable import Ecosia
+
+final class AuthStateActionTests: XCTestCase {
```

Comment:
I somehow feel this is a bit much to test a `struct` 🤔 There's some value like checking default values, but I wonder if most of this is also not just testing Swift inner working... No suggestion per se, just challenging if it's needed, happy to hear your reasonings.

Reply:
You're absolutely right! I completely agree. I've simplified it to focus only on the actual business logic (default timestamps, enum values, etc.).